src/scripts/update-todos.js

```diff
@@ -11,16 +11,20 @@ class Todo {
   }
 }
 
+const deleteTodo = (allTodo, event) => {
+  const yew = allTodo.filter((to) => to.description !== '');
+  storage.setItem('todos', JSON.stringify(yew));
+  generateTodo(yew);
+  event.target.parentNode.parentNode.remove();
+};
+
```

Comment:
- [ ] I think the delete event is not clear enough for the users. Let's create a button with a trash icon. When this button is pressed the task should be removed both from the screen and the array of items in the localStorage. With the current behavior whenever I select a task and then click out of it gets deleted.

- [ ] Let's try to clean the list from bugs as well. After I delete a task and I tried to add a new one, empty artifacts are created in the list. Watch the following clip for reference:

![todo_list_with_bugs](https://user-images.githubusercontent.com/79114233/144718492-700608f3-4682-4f65-ad46-5824f64c5019.gif)

[OPTIONAL] As you can see in the clip, I have to delete each task from the input before adding another one. I think you could improve the User Experience if you clean the input after each submit event.

Reply:
Hello @D4nielJ :smile: 

I have implemented the required changes. I've added the trash icon to the input field to show the user that she can delete a to-do item while editing.

I made a short screencast of how the original todo app that we are supposed to clone implement the feature.
If you can please see the screencast at the following [screencast](https://drive.google.com/file/d/1RPFf4xdfiQW6BS_TX_46kss9PFyERcrm/view?usp=sharing)

Reply:
Things are well explained clarified. Well done! :clap: :clap: :clap: